### PR TITLE
Finish Admin Tools

### PR DESCRIPTION
**Pull request by Ablesh1**
This adds:

- Title and Episode Number forms filled with edited anime data (edit anime page)
- Radio buttons checked basing on edited anime data (edit anime page)
- Remove anime button (delete anime page)
- on-click page redirect (add anime, edit anime and delete anime pages)

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -148,7 +148,46 @@ app.post('/PostEditAnime', async function (req, res) {
...
+			await connection.query('DELETE from anime WHERE aid=\'' + anime.aid + '\';'
+			);
+			return res.status(501).json({err: 'Anime edited'});
+		} else {
+			console.log('Anime already removed');
+			console.log('/DeleteAnime');
+			return res.status(400).json({err: 'Anime already removed'});
+		}
+	} catch (error) {
+		console.log('/DeleteAnime Error');
```
Comment (Ablesh1) [inline server.js:190]: You were supposed to add it as /PostDeleteAnime

File: server.js [hunk 2]
```diff
@@ -148,7 +148,46 @@ app.post('/PostEditAnime', async function (req, res) {
 			return res.status(400).json({err: 'Title is missing'});
 		}
 	} catch (error) {
-		console.log('/PostAddAnime Error');
+		console.log('/EditAnime Error');
+		return res.status(501);
+	}
+});
+
```
Comment (Ablesh1) [inline server.js:155]: There is a lack of protection against entering the wrong number of episodes and an error code message (also in /PostAddAnime).

File: server.js (hunk 1 above)
Comment (Ablesh1) [reply server.js:190]: I fixed it

**Review by LeMeTwo — APPROVED** [review]
Jest ok

**Pull request merged**